src/Services/Core/WB.Services.Infrastructure/WB.Services.Infrastructure.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>

```

Comment:
why switch to net6.0 from netstandard?

Reply:
MS is moving away from netstandart - it's slower and platform dependent.
"Microsoft.EntityFrameworkCore" Version="6.0.3" doesn't support netstandart